src/lib/ui/thermostat/ClimateSettingStatus.tsx

```diff
@@ -0,0 +1,76 @@
+import type { ClimateSetting } from 'seamapi'
+
+import { ThermostatCoolIcon } from 'lib/icons/ThermostatCool.js'
+import { ThermostatHeatIcon } from 'lib/icons/ThermostatHeat.js'
+import { ThermostatHeatCoolIcon } from 'lib/icons/ThermostatHeatCool.js'
+import { ThermostatOffIcon } from 'lib/icons/ThermostatOff.js'
+
+interface ClimateSettingStatusProps {
+  setting: ClimateSetting
+  iconPlacement?: 'left' | 'right'
+}
+
+export function ClimateSettingStatus({
+  setting,
+  iconPlacement = 'left',
+}: ClimateSettingStatusProps): JSX.Element {
+  return (
+    <div className='seam-climate-setting-status'>
+      {iconPlacement === 'left' && (
+        <ClimateSettingIcon mode={setting.hvac_mode_setting} />
+      )}
+      <Content
+        mode={setting.hvac_mode_setting}
+        coolingSetPoint={setting.cooling_set_point_fahrenheit}
+        heatingSetPoint={setting.heating_set_point_fahrenheit}
+      />
+      {iconPlacement === 'right' && (
+        <ClimateSettingIcon mode={setting.hvac_mode_setting} />
+      )}
+    </div>
+  )
+}
+
+function ClimateSettingIcon(props: {
+  mode: ClimateSetting['hvac_mode_setting']
+}): JSX.Element | null {
+  const { mode } = props
+
+  return (
+    <div className='seam-climate-setting-status-icon'>
+      {mode === 'cool' && <ThermostatCoolIcon />}
+      {mode === 'heat' && <ThermostatHeatIcon />}
+      {mode === 'heatcool' && <ThermostatHeatCoolIcon />}
+      {mode === 'off' && <ThermostatOffIcon />}
+    </div>
+  )
+}
+
+function Content(props: {
+  mode: ClimateSetting['hvac_mode_setting']
+  coolingSetPoint: ClimateSetting['cooling_set_point_fahrenheit']
+  heatingSetPoint: ClimateSetting['cooling_set_point_fahrenheit']
+}): JSX.Element | null {
+  const { mode, coolingSetPoint, heatingSetPoint } = props
+  const hasCoolingSetPoint = coolingSetPoint !== undefined
+  const hasHeatingSetPoint = heatingSetPoint !== undefined
+
+  if (mode === 'cool' && hasCoolingSetPoint)
+    return <span>{`${coolingSetPoint}${t.degree}F`}</span>
+
+  if (mode === 'heat' && hasHeatingSetPoint)
+    return <span>{`${heatingSetPoint}${t.degree}F`}</span>
+
+  if (mode === 'heatcool' && hasHeatingSetPoint && hasCoolingSetPoint)
+    return (
+      <span>{`${heatingSetPoint}${t.degree}F - ${coolingSetPoint}${t.degree}F`}</span>
+    )
+
+  if (mode === 'off') return <span>Off</span>
+
+  return null
+}
+
+const t = {
+  degree: `\u00b0`,
```

Comment:
Can you humor me and try
```suggestion
  degreeFahrenheit: '°F',
```

Reply:
I went with unicode since it's easier to type

Reply:
But yeah let's have it be `degreeFahrenheit`, that pair is basically one symbol

Reply:
https://github.com/seamapi/react/pull/318/commits/6a5367f6509a055bb92d8f82b35d0318004fae15#diff-4641e0e43070de61b08db5514c613b9c7438fd8d64752016e2f3cce89f52abd3R84-R88